locales/de/LC_MESSAGES/volto.po

```diff
@@ -1743,9 +1743,9 @@ msgid "Ok"
 msgstr "OK"
 
 #: components/manage/Widgets/IdWidget
-# defaultMessage: Only lowercase alphanumeric without accents, numbers, _ and - are allowed
-msgid "Only lowercase alphanumeric without accents, numbers, _ and - are allowed"
-msgstr ""
+# defaultMessage: Only lowercase letters (a-z) without accents, numbers (0-9), and the two characters "-" and "_" are allowed.
+msgid "Only lowercase letters (a-z) without accents, numbers (0-9), and the two characters "-" and "_" are allowed."
+msgstr "Nur Kleinbuchstaben (A-Z) ohne Akzente, Zahlen (0-9) und die beiden Zeichen "-" und "_" sind erlaubt."
```

Comment:
@robgietema I'd replace "Akzente" with "Umlaute" here. Geman does not have accents, just äüö